### PR TITLE
2.1.2 Adding a layer to QGEP (new chapter)

### PR DESCRIPTION
**Pull request by sjib**

### CODE REVIEW

**Comment by sjib** [comment]
Solves https://github.com/QGEP/QGEP/issues/300

**Comment by sjib** [comment]
Add your own OID in the project - solves https://github.com/QGEP/QGEP/issues/296

File: en/user-guide/layerexplanations/addviewlayer.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Add View Layer
+=================
+
+* To add a layer based on a view in the usual Add Layer dialog will give the following message, when selecting a view
+
+.. figure:: images/add_layer.png
+
+* To avoid this select the view layer and then scroll to the column "feature id" and pick "obj_id" as the relevant feature id, as QQGIS will not automatically recognize this.
```
Comment (m-kuhn) [inline en/user-guide/layerexplanations/addviewlayer.rst]: typo: QQGIS

File: en/installation-guide/workstation.rst [hunk 2]
```diff
@@ -148,3 +148,22 @@ Working with more than one database
...
+
+You have to add your OID data in the table is_oid_prefixes:
+
+https://github.com/QGEP/datamodel/blob/master/02_oid_generation.sql
+
+-- sample entry for the City of Uster - you need to adapt this entry later for your own organization
+INSERT INTO qgep.is_oid_prefixes (prefix,organization,active) VALUES ('00000000','Invalid',TRUE);
+INSERT INTO qgep.is_oid_prefixes (prefix,organization,active) VALUES ('ch11h8mw','Stadt Uster',FALSE);
+INSERT INTO qgep.is_oid_prefixes (prefix,organization,active) VALUES ('ch15z36d','SIGE',FALSE);
+INSERT INTO qgep.is_oid_prefixes (prefix,organization,active) VALUES ('ch13p7mz','Arbon',FALSE);
```
Comment (m-kuhn) [inline en/installation-guide/workstation.rst]: This should be preformatted text (code) for a nicer presentation

**Comment by m-kuhn** [comment]
Very valuable docs!

File: en/user-guide/layerexplanations/addviewlayer.rst (hunk 1 above)
Comment (sjib) [reply en/user-guide/layerexplanations/addviewlayer.rst]: ok

File: en/installation-guide/workstation.rst (hunk 2 above)
Comment (m-kuhn) [reply en/installation-guide/workstation.rst]: I think the block below needs to be indented by 2 spaces.

See http://docutils.sourceforge.net/docs/ref/rst/directives.html#code

Comment (m-kuhn) [reply en/installation-guide/workstation.rst]: Oh, and it's actually `sql`, not `ini` ;)

**Comment by sjib** [comment]
Layout corrected- @m-kuhn you can commit. Thanks

File: en/installation-guide/workstation.rst [hunk 3]
```diff
@@ -161,12 +161,12 @@ Add your own OID in the project
 
 https://github.com/QGEP/datamodel/blob/master/02_oid_generation.sql
 
-::
+.. code:: ini
```
Comment (m-kuhn) [inline en/installation-guide/workstation.rst]: One last thing

    s/ini/sql/

**Comment by sjib** [comment]
It is sql Code – so +.. code:: sql would be correct?

 

 

Von: Matthias Kuhn [mailto:notifications@github.com] 
Gesendet: Mittwoch, 30. August 2017 16:27
An: QGEP/docs
Cc: Stefan; Author
Betreff: Re: [QGEP/docs] 2.1.2 Adding a layer to QGEP (new chapter) (#45)

 

@m-kuhn commented on this pull request.

  _____  

In en/installation-guide/workstation.rst <https://github.com/QGEP/docs/pull/45#discussion_r136084046> :

> @@ -161,12 +161,12 @@ Add your own OID in the project
 
 https://github.com/QGEP/datamodel/blob/master/02_oid_generation.sql
 
-::
+.. code:: ini

One last thing

s/ini/sql/

**Comment by m-kuhn** [comment]
Exactly

**Comment by sjib** [comment]
Hope it is ok now.

**Comment by m-kuhn** [comment]
Yes, looks good

**Pull request merged**